estate/models/estate_property.py

```diff
@@ -0,0 +1,19 @@
+from odoo import models, fields
+
```

Comment:
You should give one more space above a class definition, according to "PEP 8: E302 expected 2 blank lines, found 1"